content/docs/advanced/nav.yml

```diff
@@ -22,30 +22,26 @@
       href: /advanced/customquery.html
       forceInternal: true
       subitems:
-        - id: data-aware-ui-components
+        - id: when-aware-ui-components
```

Comment:
Not sure if this title makes sense?

Reply:
Should be `data` instead of `when`, thanks will update and check this once again